src/update/balrog.cpp

```diff
@@ -54,8 +54,9 @@ constexpr const char* BALROG_MACOS_UA = "Darwin_x86";
 #  error Platform not supported yet
 #endif
 
-constexpr const char* BALROG_CERT_SUBJECT_CN =
-    "aus.content-signature.mozilla.org";
+// TODO - Do we need to do this cert subject verification?
+//constexpr const char* BALROG_CERT_SUBJECT_CN =
+//    "aus.content-signature.mozilla.org";
```

Comment:
@bakulf - do we need to do the cert subject verification. Code was:

```
  const QSslCertificate& leaf = list.constFirst();
  logger.log() << "Validating cert subject";
  QStringList cnList = leaf.subjectInfo("CN");
  if (cnList.isEmpty() || cnList[0] != BALROG_CERT_SUBJECT_CN) {
    logger.log() << "Invalid CN:" << cnList;
    return false;
  }
```

If yes, I suggest we also hand this work off to the balrog go module and I can write a small helper function to take care of this, which we can pass the `BALROG_CERT_SUBJECT_CN` to keep the balrog go module very agnostic.

If no, I'll push another small commit to remove these commented codes.

Reply:
autograph doesn't implement as its expectation is a closed loop. It verifies the chain all matches, but doesn't do a check like this on the leaf.